build.gradle

```diff
@@ -3,9 +3,11 @@ import me.modmuss50.mpp.ReleaseType
 plugins {
     id "architectury-plugin" version "3.4-SNAPSHOT"
     id "dev.architectury.loom" version "1.3-SNAPSHOT" apply false
-    id "me.modmuss50.mod-publish-plugin" version "0.4.5"
+    id "me.modmuss50.mod-publish-plugin" version "0.5.1"
```

Comment:
8.4 is latest iirc, shouldn't require any other changes to the config

Reply:
nm, newer version of gradle needed (was using 8.4, now 8.10)